apps/backend/src/datasources/postgres/ProposalDataSource.ts

```diff
@@ -1073,6 +1077,16 @@ export default class PostgresProposalDataSource implements ProposalDataSource {
   }
 
   async doesProposalNeedTechReview(proposalPk: number): Promise<boolean> {
+    const getWorkflowStatus = (

```

Comment:
Maybe it is better to name this just `workflowStatus` instead of using prefix get because it contains the status and not a function to get it.